### PR TITLE
Implemented some test files and jsonable functions in classes

### PR DESCRIPTION
**Pull request by jeremiahwong007**

### CODE REVIEW

File: src/entities/Singleton.java [hunk 1]
```diff
@@ -183,4 +186,17 @@ public void setAvailability(boolean availability) {
...
 
 
+    @Override
+    public JSONObject jsonify() {
+        JSONObject final_data = new JSONObject();
+        final_data.put("id", this.id);
+        final_data.put("price", this.price);
+        final_data.put("name", this.name);
+        final_data.put("description", this.description);
+        final_data.put("allowedAddonTypes", this.allowedAddonTypes);
```
Comment (VijayS02) [inline src/entities/Singleton.java:196]: These need to be jsonified

Comment (VijayS02) [reply src/entities/Singleton.java:196]: You cant directly put objects into a json object, use the jsonify method

**Review by VijayS02 — CHANGES_REQUESTED** [review]

File: src/entities/Singleton.java (hunk 1 above)
Comment (jeremiahwong007) [reply src/entities/Singleton.java:196]: Screenshot 2021-12-02 at 12 15 01 PM

Comment (jeremiahwong007) [reply src/entities/Singleton.java:196]: Is this what you mean?

**Pull request closed without merging**